tests/test_double_blind_conference.py

```diff
@@ -2096,9 +2096,9 @@ def test_release_accepted_notes(self, client, request_page, selenium):
         assert tabs
         with pytest.raises(NoSuchElementException):
             notes_panel.find_element_by_class_name('spinner-container')
-        assert tabs.find_element_by_id('accepted-poster-papers')
+        # assert tabs.find_element_by_id('accepted-poster-papers')
```

Comment:
what about this assertion? can you enable it again?

Reply:
yes! I thought I had, sorry